docs-src/docs/dataset-refinement/prediction-format.md

```diff
@@ -0,0 +1,188 @@
+---
+title: The prediction format
+---
+
+Predictions use the OpenLabel format/schema. This is the same format as the one used
+for [uploading pre-annotations](../kognic-io/pre_annotations/#openlabel-support). General information about the
+OpenLabel format can be found in [here](../openlabel/openlabel-format).
+
+## Supported prediction features
+
+:::note
+Only one type of feature per prediction is supported.
+:::
+
+The current API for uploading predictions supports the following features:
+
+| Name         | OpenLABEL field | Description        |
+|--------------|-----------------|--------------------|
+| Cuboid       | `cuboid`        | Cuboid in 3D       |
+| Bounding box | `bbox`          | Bounding box in 2D |
+
+Note that all geometries should be specified under frames rather than in the root of the pre-annotation. The rotation of
```

Comment:
Mix geometries and features. Maybe geometries is the best?

Reply:
Good point, will change to geometries